cf-monitord/mon_cpu.c

```diff
@@ -50,7 +50,7 @@ void MonCPUGatherData(double *cf_this)
 
     if ((fp = fopen("/proc/stat", "r")) == NULL)
     {
-        Log(LOG_LEVEL_VERBOSE, "Didn't find proc data");
+        Log(LOG_LEVEL_ERR, "Could not open /proc/stat while gathering CPU data (fopen: %s)", GetErrorStr());
```

Comment:
I believe this code is shared with systems that don't have ``/proc/stat`` so it should not be an error not to be able to open it.